README.md

```diff
@@ -110,6 +110,14 @@ pip install scalene
 scalene opentelemetry-<PACKAGE>/tests/performance/resource-usage/<PATH_TO_TEST>/profile_resource_usage_<NAME_OF_TEST>.py
 ```
 
+## Python Version Support Addition and Removal
+
+This project supports the latest Python versions. As new Python versions are released, support for them is added and
+as Python versions are deprecated, support for them is removed.
+
+We add support for new Python versions as soon as possible after they become stable.
```

Comment:
Would be good to give a rough timeline here. as soon as possible means different things for different people :D

Maybe within 3 months of stability? That seems reasonable

Reply:
That sounds reasonable, I have added that as well :v: